rootfs/usr/share/www/static/styles.css

```
@@ -0,0 +1,147 @@
+@font-face {
+  font-family: "Roboto";
+  src: local("Roboto Regular"), local("Roboto-Regular"),
+    url(/landingpage/static/Roboto-Regular.woff2) format("woff2");
+  font-weight: 400;
+  font-style: normal;
+}
+
+html {
+  color: var(--primary-text-color, #212121);
+  background-color: #0277bd !important;
```

No need to be important here. That was only because Home Assistant was applying their styles to the element.
```suggestion
  background-color: #0277bd;
```